pcsx2/CDVD/macOS/DriveUtility.mm

```diff
@@ -0,0 +1,91 @@
+/*  PCSX2 - PS2 Emulator for PCs
+*  Copyright (C) 2002-2020  PCSX2 Dev Team
+*
+*  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+*  of the GNU Lesser General Public License as published by the Free Software Found-
+*  ation, either version 3 of the License, or (at your option) any later version.
+*
+*  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+*  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+*  PURPOSE.  See the GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License along with PCSX2.
+*  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// guard BOOL as Obj-C need to typedef it
+#define BOOL _BOOL
+#include "PrecompiledHeader.h"
+#include "../CDVDdiscReader.h"
+#undef BOOL
+
+#import <Foundation/Foundation.h>
+
+#include <IOKit/storage/IOMedia.h>
+#include <IOKit/storage/IOCDMedia.h>
+#include <IOKit/storage/IOCDTypes.h>
+#include <IOKit/storage/IODVDMedia.h>
+#include <IOKit/IOBSD.h>
+
+std::vector<std::string> GetDriveListFromClasses(CFMutableDictionaryRef classes) {
+    io_iterator_t iterator = IO_OBJECT_NULL;
+    kern_return_t result;
```

Comment:
Use tabs, not spaces. Consider using the included clang-format to reformat your PR